scripts/shared/is-tui-package-name.ts

```diff
@@ -1,5 +1,5 @@
-export function isTuiPackageName(name = '', ignores: readonly string[] = []): boolean {
+export function isTuiPackageName(name?: string, ignore: readonly string[] = []): boolean {
```

Comment:
This is unnecessary, it was fine as it was, `?.` could just be removed as `name` is always a `string`

Reply:
fixed